tests/smtlib/testfile-steps-bound1.dolmen.err.expected

```diff
@@ -0,0 +1 @@
+(error "Invalid action during mode Util.Assert: Set option steps_bound")
```

Comment:
"mode Util.Assert" is not very nice; if that's what `@@deriving show` does we should write our own printer instead.